src/status_im2/common/toasts/view.cljs

```diff
@@ -0,0 +1,104 @@
+(ns status-im2.common.toasts.view
+  (:require
+   [quo2.core               :as quo2]
+   [react-native.core       :as rn]
+   [react-native.gesture    :as gesture]
+   [react-native.reanimated :as reanimated]
+   [reagent.core            :as reagent]
+   [status-im.utils.utils   :as utils.utils]
+   [utils.re-frame          :as rf]))
+
+(def ^:private slide-out-up-animation
+  (-> reanimated/slide-out-up-animation
+      .springify
+      (.damping 20)
+      (.stiffness 300)))
+
+(def ^:private slide-in-up-animation
+  (-> reanimated/slide-in-up-animation
+      .springify
+      (.damping 20)
+      (.stiffness 300)))
+
+(def ^:private linear-transition
+  (-> reanimated/linear-transition
+      .springify
+      (.damping 20)
+      (.stiffness 300)))
+
+(defn container
+  [id]
+  (let [dismissed-locally? (reagent/atom false)
+        close!             #(rf/dispatch [:toasts/close id])
+        timer              (reagent/atom nil)
+        clear-timer        #(utils.utils/clear-timeout @timer)]
```

Comment:
we have `react-native.background-timer`

Reply:
this one is already using background timer